pkg/apis/core/v1beta1/defaults.go

```diff
@@ -128,21 +128,10 @@ func SetDefaults_Shoot(obj *Shoot) {
 	}
 
 	for i, worker := range obj.Spec.Provider.Workers {
-		kubernetesVersion := obj.Spec.Kubernetes.Version
-		if worker.Kubernetes != nil && worker.Kubernetes.Version != nil {
-			kubernetesVersion = *worker.Kubernetes.Version
-		}
-
```

Comment:
Why  did you remove this? Maybe add a unit test for it?

Reply:
It was used to check if the version is >= 1.22, now it is always the case.

Reply:
Ah right, I see it now. Thanks :)